compliance-monitor/monitor.py

```diff
@@ -7,6 +7,7 @@
 from shutil import which
 from subprocess import run
 from tempfile import NamedTemporaryFile
+from _thread import allocate_lock, get_ident
```

Comment:
Why the low-level API and not `threading.Lock` etc?

Reply:
b18e4c3ae5f7b6e8d0afbe21fc5c587ce8263d43